docs/src/administration/users.md

```diff
@@ -3,136 +3,182 @@ title: "User administration"
+### Add a user to a project
 
-After inviting a new user, users without a Platform.sh account receive an invitation email asking to confirm their details and register an account.
+To add a user to a project, follow these steps:
 
-In order to push and pull code (or to SSH into one of the project's environments), the user needs to add an SSH key or use the Platform CLI.
+{{< codetabs >}}
+---
+title=In the console
+file=none
+highlight=false
+---
 
-If the user already has an account, they receive an email with a link to access the project.
+1. In the console, select the project where you want to add a new user.
+2. Under **Settings**, click **Access**.
+3. Click **+ Add**
+4. Add the user's details and choose their permissions.
+5. Click **Save**.
 
-## Manage users permissions with the CLI
+{{< note theme="warning">}}
 
-You can use the [Platform.sh CLI (Command Line Interface)](/development/cli/_index.md). to fully manage your users and integrate with any automated system.
+If you are an organization owner, or an organization user with [manage plan](/administration/organizations.md#organization-permissions) or [manage users](/administration/organizations.md#organization-permissions) permissions wanting to add a user to a project or an environment, the user has to create an account before they can contribute to the project.
 
-Available commands:
+{{< /note >}}
+
+<--->
+---
+title=Using the CLI
+file=none
+highlight=false
+---
+
+You can use the [Platform.sh CLI (Command Line Interface)](/development/cli/_index.md) to create new users.
 
 * `platform user:add`
-  * Add a user to the project
-* `platform user:delete`
-  * Delete a user
-* `platform user:list` (`users`)
-  * List project users
-* `platform user:role`
-  * View or change a user's role
 
-For example, the following command would add `alice@example.com` with **Project Admin** role in the current project.
+Say you want to add `user1@example.com` to the project with **Project Admin** permissions:
 
 ```bash
-platform user:add alice@example.com -r admin
+platform user:add user1@example.com -r admin
 ```
 
-After adding `alice@example.com`, Alice will receive an invitation email asking to confirm her details and optionally, register for a Platform.sh account.
+{{< /codetabs >}}
 
-To give Bob different levels of access depending on the environment type, in the current project:
- 
--  **Viewer** role to the **Production environment**
--  **Contributor** role to **all Staging type environments**
--  **Admin** role to **all Development type environments**
+Once you add a user to a project, they receive an email with instructions.
+For SSH access changes to apply after you add a user to a project, you have to redeploy each environment by either clicking **Redeploy** in the console or running `platform redeploy`.
 
-You would run:
+## Manage existing users
```

Comment:
I would say I feel it should be changed for a few reasons:
* Managing users is a superset of managing existing users. Managing users includes both new and existing users, so putting those two things on the same level seems a little confusing. Otherwise, I'm not sure what else would fall under managing users (other than adding new users).
* Managing existing users exists on the same level as adding new users. That's what the initial confusion is for me -- they are two subsets of the same topic.
* With the overall structure of the article, it makes sense to me to have a section that's about the practical aspect of managing users. I think separating that from the reference sections above makes sense. That distinction becomes less clear when the section is split into multiple sections.